addons/project/tests/test_access_rights.py

```diff
@@ -366,11 +366,6 @@ def test_project_user_crud_own_private_task(self):
         self.assertEqual(vals[0]['id'], private_task.id)
         self.assertEqual(vals[0]['name'], private_task.name)
 
-    @users('Project user')
-    def test_project_user_cannot_create_private_task_for_another_user(self):
-        with self.assertRaises(AccessError):
-            self.create_private_task('test private for another user', self.env.user, user_ids=[Command.set(self.user_projectuser.ids)])
-
```

Comment:
Instead of removing the test, I would check that it is now feasible. I would also test the feature you just code which is that on private task, the current user is added to the list of assignees. I would also test that your feature is not activated when`project_id` or `default_project_id` is provided.
Thanks 🙂